libs/ml/tests/unit/graph/graph.cpp

```diff
@@ -1524,7 +1524,7 @@ TYPED_TEST(GraphTest, graph_charge_backward_conv_dense)
 
   OperationsCount const charge = g.ChargeBackward(dense);
   OperationsCount const expected_charge =
-      9044198;  // Pre-calculated backward charge for give shape.
+      2319284827366;  // Pre-calculated backward charge for give shape.
```

Comment:
Isn't that value too big? 38 minutes of execution time.

Reply:
that looks like an old value, which should be: 193275889546
about 3 minutes